test/util/RandomTestHelpers.h

```diff
@@ -8,11 +8,14 @@
 #include <cstddef>
 #include <random>
 
+#include "util/Random.h"
+
 // Create a array of non-deterministic random seeds for use with random number
 // generators.
 template <size_t NumSeeds>
-static std::array<unsigned int, NumSeeds> createArrayOfRandomSeeds() {
-  std::array<unsigned int, NumSeeds> seeds{};
-  std::ranges::generate(seeds, []() { return std::random_device{}(); });
+static std::array<Seed, NumSeeds> createArrayOfRandomSeeds() {
```

Comment:
```suggestion
std::array<Seed, NumSeeds> createArrayOfRandomSeeds() {
```

Static functions in a header file are typically not what you intended, although it doesn't make much of a difference here.

Reply:
Whoops. Fixed in [1afd7f7](https://github.com/ad-freiburg/qlever/pull/1126/commits/1afd7f7eb12a6341e6209c36077262178bfb5086).